pkg/process/net/windows_pipe.go

```diff
@@ -8,7 +8,26 @@
 package net
 
 import (
+	"fmt"
 	"net"
+	"time"
+
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+	"github.com/Microsoft/go-winio"
+)
+
+const (
+	// Buffer sizes for the system probe named pipe.
+	// The sizes are advisory, Windows can adjust them, but should be small enough to preserve
+	// the nonpaged pool.
+	namedPipeInputBufferSize  = int32(4096)
+	namedPipeOutputBufferSize = int32(4096)
+
+	// DACL for the system probe named pipe.
+	// SE_DACL_PROTECTED (P), SE_DACL_AUTO_INHERITED (AI)
+	// Allow Everyone (WD)
+	// nolint:revive // TODO: Hardened DACL and ensure the datadogagent run-as user is allowed.
```

Comment:
Why adding `nolint:revive`?

Reply:
The linter gives a warning about TODO comments for unknown reasons.  I copied this convention from existing code.